SPECS/frr/frr.spec

```diff
@@ -18,6 +18,8 @@ Patch3:         0003-fips-mode.patch
 Patch4:         0004-remove-grpc-test.patch
 Patch5:         CVE-2023-46752.patch
 Patch6:         CVE-2023-46753.patch
+Patch7:			CVE-2023-47235.patch
+Patch8:			CVE-2023-47234.patch
```

Comment:
Nit-pick: can you align the new patches with the other ones and replace tabs with spaces?

Reply:
Fixed! Good Catch.